bower.json

```diff
@@ -1,37 +1,45 @@
 {
-  "name": "purescript-grid-reactors",
```

Comment:
The churn in this file is due to `spago bump-version` replacing the `bower.json`.